client/yarn.lock

```diff
@@ -14605,10 +14668,10 @@ webpack-dev-server@^3.1.14:
     webpack-log "^2.0.0"
     yargs "12.0.2"
 
-webpack-dev-server@^3.10.1:
-  version "3.10.1"
-  resolved "https://registry.yarnpkg.com/webpack-dev-server/-/webpack-dev-server-3.10.1.tgz#1ff3e5cccf8e0897aa3f5909c654e623f69b1c0e"
-  integrity sha512-AGG4+XrrXn4rbZUueyNrQgO4KGnol+0wm3MPdqGLmmA+NofZl3blZQKxZ9BND6RDNuvAK9OMYClhjOSnxpWRoA==
+webpack-dev-server@^3.9.0:
```

Comment:
Is this right?

Reply:
Might be a bad merge, but it's not correct -- good catch.  The package.json is incorrect as well, with a couple duplicated dependencies from devDependencies.

Reply:
@dannon you're right. I will remove webpack-dev-server from package.json:dependencies and force-push.